src/Symfony/Component/Workflow/Event/EventNameTrait.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Event;
+
+use Symfony\Component\Workflow\Exception\InvalidArgumentException;
+
+/**
+ * @author Nicolas Rigaud <squrious@protonmail.com>
+ */
+trait EventNameTrait
```

Comment:
I suggest marking the trait as `@internal`. I don't see a reason to make it reusable as a public API.